docs/learn/dapp-staking/dapp-staking-protocol.md

```diff
@@ -320,6 +320,38 @@ others will be left out. There is no strict rule which defines this behavior - i
 having a larger stake than the other dApp(s). Technically, at the moment, the dApp with the lower `dApp Id` will have the advantage over a dApp with
 the larger Id but this can change in the future.
 
+### Tier ranking system
+Because dApps at the same tier receive equal rewards regardless of their staked amount, we introduced a ranking system for dApps within the tier itself. This will improve reward distribution for dApps that perform better within a tier if there are available rewards to be distributed.
```

Comment:
```suggestion
Because dApps at the same tier receive equal rewards regardless of their staked amount, ranking system has been introduced for dApps within the tier itself. This will improve reward distribution for dApps that perform better within a tier if there are available rewards to be distributed.
```

_we_ approach isn't used anywhere in the docs, hence the suggestion.